spec/lib/common/models/redis_store_spec.rb

```diff
@@ -62,39 +65,48 @@
 
   describe '#save' do
     it 'saves serialized class to redis with the correct namespace' do
-      expect_any_instance_of(Redis).to receive(:set).once.with(
-        'my_namespace:e66fd7b7-94e0-4748-8063-283f55efb0ea',
-        '{":uuid":"e66fd7b7-94e0-4748-8063-283f55efb0ea",":email":"foo@bar.com"}'
-      )
+      expected_key = 'my_namespace:e66fd7b7-94e0-4748-8063-283f55efb0ea'
+      expected_val = '{":uuid":"e66fd7b7-94e0-4748-8063-283f55efb0ea",":email":"foo@bar.com"}'
+
+      expect(Redis.current).to receive(:set).once.with(expected_key, expected_val)
+      expect(SECONDARY_REDIS).to receive(:set).once.with(expected_key, expected_val)
       subject.save
     end
   end
 
   describe '#update' do
     it 'updates only user the user attributes passed in as arguments' do
-      expect(subject).to receive(:save).once
```

Comment:
Had to remove this because this implicitly creates a method stub for `:save` - which meant that the data was never actually stored into Redis... which is a problem because I added an expectation below where I read directly from Redis.